.github/workflows/detect_version_bump.yml

```diff
@@ -21,10 +21,16 @@ jobs:
         run: |
           NEW_VERSION=$(echo ${{ github.head_ref }} | cut -d _ -f4 )
           echo "::set-output name=version::$NEW_VERSION"
+      - name: Is prerelease?
+        id: is-prerelease
+        run: |
+          IS_PRERELEASE=$([[ "${{ steps.get-new-version.outputs.version }}" == *-* ]] && echo true || echo false)
```

Comment:
I _believe_ this will capture whether or not the new version is a prerelease version.

⚠️ It assumes that all prerelease versions will match the unix pattern `*-*` (in other words, any prerelease version will contain a `-` and non-prerelease versions will never have a `-` in them).

❓ Is this ^ a valid assumption?

Reply:
That should be correct based on the [bumpversion.cfg](https://github.com/wayfair-incubator/columbo/blob/29edbcb852defeb6ceb5ba73b68727b2aa99552d/.bumpversion.cfg#L5). However, you could change the pattern to more closely match that config to reduce the chance of a false positive.

For working on CI things, I used a [testing repo](https://github.com/plannigan/actions-playground/) to verify that code executed as expected. Might be a good idea to give it a dry run instead of use finding out there is an issue when we try to do a real release.

Reply:
Thanks for the review and feedback! I'll refine the pattern matching a bit and test it locally before merging.